### PR TITLE
Remove unused method

### PR DESCRIPTION
**Pull request by delphaber**

### CODE REVIEW

**Comment by tute** [comment]
Won't Rails run `bundle install` before suspender does, running it twice?

**Comment by drapergeek** [comment]
Yes I think this method is called by the normal rails app generator and we're just putting it off until later.

**Comment by croaky** [comment]
Yes, I think the intent of this method to prevent Bundler from running twice. If this commit were to be merged, I think we'd revert to that, slower behavior.

We invoke `bundle install` at the moment we want it, inside the `customize_gemfile` method, on line 61 of `lib/suspenders/generators/app_generator.rb`

File: lib/suspenders/generators/app_generator.rb [hunk 1]
```diff
@@ -212,10 +212,6 @@ def outro
       say "Remember to run 'rails generate airbrake' with your API key."
     end
 
-    def run_bundle
-      # Let's not: We'll bundle manually at the right spot
-    end
-
```
Comment (bf4) [inline lib/suspenders/generators/app_generator.rb]: I think this is meant to over-write `run_bundle` defined in `app_base`

- https://github.com/rails/rails/blob/4-2-stable/railties/lib/rails/generators/app_base.rb#L334
- https://github.com/rails/rails/blob/4-2-stable/railties/lib/rails/generators/rails/app/app_generator.rb#L259

Although I don't know why you don't just set skip_bundle to true https://github.com/rails/rails/blob/4-2-stable/railties/lib/rails/generators/app_base.rb#L32

**Comment by delphaber** [comment]
Oh I'm sorry. Didn't think about inheritance,

**Comment by tute** [comment]
If you'd like to update the PR, `skip_bundle` could be an improvement, making the code clearer IMO.

**Comment by croaky** [comment]
Agreed on `skip_bundle`. Clearer, more intentional API.

**Comment by delphaber** [comment]
Ok I'm gonna update it :)

**Comment by delphaber** [comment]
Updated :)

**Comment by mcmire** [comment]
:+1:

**Comment by croaky** [comment]
Thanks!

**Pull request merged**